config/rbac/aws_data_gather.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: aws-data-gather
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: aws-data-gather
+rules:
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
```

Comment:
- I think the resources here are inadequate as I see `pod: get, cm: get` in code
- Easy one would be to add `- pod` here or separate `pod` roles